File: excel/jruby/read/excel_read.rb
#! /usr/bin/jruby
# -*- coding: utf-8 -*-
#
#	excel_read.rb
#					Aug/08/2013
#
# --------------------------------------------------------
load '/var/www/data_base/common/ruby_common/text_manipulate.rb'
load '/var/www/data_base/common/jruby_common/excel_manipulate.rb'
# --------------------------------------------------------
puts "*** 開始 ***"
excel_file=ARGV[0]
#
dict_aa = excel_read_proc(excel_file)

dict_display_proc(dict_aa)

puts "*** 終了 ***"
# --------------------------------------------------------

File: excel/rhino/read/excel_read.js
#! /usr/bin/java org.mozilla.javascript.tools.shell.Main
// ----------------------------------------------------------------
/*
	excel_read.js
					Aug/08/2013

*/
// ----------------------------------------------------------------
load ("/var/www/data_base/common/rhino_common/text_manipulate.js");
load ("/var/www/data_base/common/rhino_common/excel_manipulate.js");

// ----------------------------------------------------------------
print ("*** 開始 ***");

var excel_file = arguments[0];

var dict_aa = excel_read_proc (excel_file);

dict_display_proc (dict_aa);

print ("*** 終了 ***");  
  
// ----------------------------------------------------------------
